tests/integration/test_api_gateway.py

```diff
@@ -689,3 +689,106 @@ def test_get_model_with_invalid_name(self):
 
         except ClientError as e:
             self.assertEqual(e.response['Error']['Code'], 'NotFoundException')
+
+    def test_put_integration_dynamodb_proxy_validation_without_response_template(self):
```

Comment:
The two test function seem to share a lot of code in common. If there's an easy way to extract the common parts, that would be great. If too much work, then probably not worth the effort and we can merge as is (i.e., not critical). Thanks